Lua/Lua.sublime-syntax

```diff
@@ -82,6 +87,20 @@ contexts:
     - include: end
     - include: statements
 
+  shebang:
+    - match: ^\#!
+      scope: punctuation.definition.comment.lua
+      set: shebang-body
+    - match: ''
```

Comment:
Immediatelly popping will cause issues once we decide to embed LUA into Markdown for instance.

![grafik](https://user-images.githubusercontent.com/16542113/122636150-fcaab200-d0e7-11eb-9420-c6b11303711b.png)

That's why Bash uses

```
    - match: ^|(?=\S)
      pop: true
```

Reply:
Interesting knowing!